samples/augment-reality-to-show-hidden-infrastructure/src/main/java/com/esri/arcgismaps/sample/augmentrealitytoshowhiddeninfrastructure/screens/AugmentedRealityScreen.kt

```diff
@@ -0,0 +1,369 @@
+import androidx.compose.material3.Button
+import androidx.compose.material3.Card
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.LinkAnnotation
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.TextLinkStyles
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withLink
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.core.content.edit
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.LoadStatus
+import com.arcgismaps.toolkit.ar.WorldScaleSceneView
+import com.arcgismaps.toolkit.ar.WorldScaleSceneViewStatus
+import com.arcgismaps.toolkit.ar.WorldScaleTrackingMode
+import com.arcgismaps.toolkit.ar.rememberWorldScaleSceneViewStatus
+import com.esri.arcgismaps.sample.augmentrealitytoshowhiddeninfrastructure.R
+import com.esri.arcgismaps.sample.augmentrealitytoshowhiddeninfrastructure.components.AugmentedRealityViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+private const val KEY_PREF_ACCEPTED_PRIVACY_INFO = "ACCEPTED_PRIVACY_INFO"
+
+@Composable
+fun AugmentedRealityScreen(
+    sampleName: String
+) {
+    val augmentedRealityViewModel: AugmentedRealityViewModel = viewModel()
+
+    var displayCalibrationView by remember { mutableStateOf(false) }
+    var initializationStatus by rememberWorldScaleSceneViewStatus()
+    var trackingMode by remember { mutableStateOf<WorldScaleTrackingMode>(WorldScaleTrackingMode.Geospatial()) }
+
+    var showDropdownMenu by remember { mutableStateOf(false) }
+    var isPipeShadowVisible by remember { mutableStateOf(true) }
+    var isLeaderVisible by remember { mutableStateOf(true) }
+
+    val sharedPreferences = LocalContext.current.getSharedPreferences("", Context.MODE_PRIVATE)
+    var acceptedPrivacyInfo by rememberSaveable {
+        mutableStateOf(
+            sharedPreferences.getBoolean(
+                KEY_PREF_ACCEPTED_PRIVACY_INFO,
+                false
+            )
+        )
+    }
```

Comment:
I would recommend using Google datastore instead of shared preferences here as we have opted to use this for storing "preferences" like favorited samples. As well as, [Google docs](https://developer.android.com/topic/libraries/architecture/datastore) mentions:
> If you're currently using [SharedPreferences](https://developer.android.com/reference/kotlin/android/content/SharedPreferences) to store data, consider migrating to DataStore instead.

Reply:
It's fine as it is, lets opt not to use shared preferences in the future 👍